hmap/rtw_hmap.h

```diff
@@ -10,15 +10,16 @@ typedef int (*rtw_hmap_cmp_fn)(const void *, const void *);
 
 typedef struct rtw_hmap_elem_s_ {
     void *key;
-    void *data;
     struct rtw_hmap_elem_s_ *next;
 } rtw_hmap_elem_;
 
 typedef struct {
     rtw_hmap_hash_fn hash_func;
     rtw_hmap_cmp_fn comp_func;
     rtw_hmap_elem_ *elements[RTW_HMAP_BUCKET_SIZE_];
+    rtw_vec keys_and_values;
     size_t value_len;
```

Comment:
We don't need to store `value_len` and `key_len` since we store `value_len + key_len` in the inner vector. I think storing `key_len` is enough.